scripts/generate_export.py

```diff
@@ -346,7 +350,7 @@ def sequences_overlap(true_seq, pred_seq):
         submission_batch = complete_submissions[batch_start:batch_end]
         # FULL WORK FLOW
         full_dfs = []
-        for submission in submission_batch:
+        for submission in complete_submissions:
```

Comment:
This should not be changed. Note how this is enclosed in another for loop above. If this run was batched, you would be running the export generation len(complete_submissions) // BATCH_SIZE times.